nuxt.config.js

```diff
@@ -12,15 +12,30 @@ export default {
                     authorization: `Token ${process.env.AUTH_TOKEN}`,
                 },
             }
-            const talks = await axios.get(
-                `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=talk,sponsored`,
-                config,
-            )
-            const tutorials = await axios.get(
-                `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=tutorial`,
-                config,
-            )
+            const talks = await axios
+                .get(
+                    `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=talk,sponsored`,
+                    config,
+                )
+                .catch((error) => {
+                    if (error.response.status === 404) {
+                        return { data: [] }
+                    }
+                })
```

Comment:
I suggest we can return empty  array on catch  instead.
Or is there any scenario that we want return undefined on catch.
```
            const talks = await axios
                .get(
                    `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=talk,sponsored`,
                    config,
                )
                .catch((error) => {
                        return { data: [] }
                })
```